src/content/concepts/entry-points.md

```diff
@@ -64,6 +65,28 @@ T> __"Scalable webpack configurations"__ are ones that can be reused and combine
 
 T> You can pass empty object `{}` to `entry` when you have only entrypoints generated by plugins.
 
+## Array Syntax
```

Comment:
Thank you for your patience.

I think the `Single Entry Syntax` section already cover String Syntax (`string`) and Array Syntax (`[string]`):

![image](https://user-images.githubusercontent.com/1091472/98443733-442b9180-2148-11eb-92c9-46a15d9bab67.png)

I'm not sure it's a good idea to separate these them. We want to underline the different usages between Single Entry and Multi Entry, instead of the String/Object/Array syntax. How about adding this example to `Single Entry Syntax` section?

We can change `What happens when you pass an array to entry? ` tip into a paragraph and add the example afterward.

Reply:
Hi , chenxsan:

Thank you for your reply, 
your suggest looks great, 
Let's do this.

Reply:
Actually , this is my first time do Pull-Request to open source community.
so , I am not really know about what is the next step?

Uh... , Do I need to cancel or delete this pull request , modify the content , then do pull request again?
Or you just modify directly, then merge.
Thanks.

Reply:
Just update the change in another commit and push to this branch. No need to cancel or delete this PR.